src-gui/src/renderer/rpc.ts

```diff
@@ -293,15 +293,21 @@ export async function updateAllNodeStatuses() {
   const network = getNetwork();
   const settings = store.getState().settings;
 
+  // We pass all electrum servers to the backend without checking them (ElectrumBalancer handles failover),
+  // but check these anyway since the status appears in the GUI
+  const blockchains = [Blockchain.Bitcoin];
```

Comment:
I think 

settings.useMoneroRpcPool ? [Blockchain.Bitcoin] : [Blockchain.Bitcoin, Blockchain.Monero]

is cleaner because it avoids mutation which I prefer but that's a nitpick.